### DISCUSSION TITLE
Missing some HTML

### DISCUSSION THREAD

**Discussion by Nixinova** [lowlighter/metrics#360] [Q&A] [answered]
These metrics, which uses indepth, only shows 114kB HTML, whereas just my website [shows 120kB](https://api.github.com/repos/nixinova/website/languages). The only emails I have used are specified in my config. I'm not sure what's missing here.

![image](https://user-images.githubusercontent.com/42429413/120878418-60c17680-c610-11eb-93bd-e27a5ea1ba50.png)

**Comment by lowlighter** [comment]
Will try to reproduce when I have some time 🙂

**Reply by lowlighter** [reply]
So not sure whether it'll change anything, but analyzers switched from `exec` to `spawn` to be able to process largers patches (since `exec` has a limited buffer, it was skipping some commits)

Though it seems that it processes more commits than actually reported by GitHub, need to investigate why

**Comment by lowlighter** [comment, marked as answer]
I'll archive this discussion snce it's kind of old, hopefully it got better since, else it'd be eventually investigated upon in #531 👍
